pages/user guides/tidal-vms.md

```diff
@@ -11,19 +11,19 @@ To help you get your Tidal Migrations discovery phase up and running, we've crea
 
 ## Ubuntu 18.04
 
-The Ubuntu-18.04-server-amd64 image comes with the latest version of [Tidal Tools](https://tidalmigrations.com/tidal-tools/) and [Machine Stats](https://pypi.org/project/machine-stats/) installed. To get started, download the Virtual Machine image from the link given below. You can import this OVA file into your VirtualBox or VMware Workstation. Use the following credentials to login.
+The Ubuntu-18.04-server-amd64 image comes with the latest version of [Tidal Tools](https://tidalmigrations.com/tidal-tools/), [Machine Stats](https://pypi.org/project/machine-stats/) and [DNS Tools](https://dnstools.ninja/) installed. To get started, download the Virtual Machine image from the link given below. You can import this OVA file into your VirtualBox or VMware Workstation. Use the following credentials to login as an user.
 
 ```
-Username: ubuntu
-Password: tidal
+Username: vagrant
```

Comment:
I see you added an item on the [checklist](https://trello.com/c/y6paldsk/3683-move-back-to-packer-vmware-approach-to-create-the-ova) to work on this change 👍

Reply:
@jmariomejiap I have moved a couple of items from this checklist to a separate card since it'll require more dev time and was only added here as a stretch goal.